zerver/lib/actions.py

```diff
@@ -1759,15 +1759,14 @@ def do_change_password(user_profile, password, commit=True,
                                  modified_user=user_profile, event_type='user_change_password',
                                  event_time=event_time)
 
-def do_change_full_name(user_profile, full_name, log=True):
-    # type: (UserProfile, Text, bool) -> None
+def do_change_full_name(user_profile, full_name, acting_user):
```

Comment:
@rishig I agree, but [here](https://github.com/zulip/zulip/blob/56cecc48911445f91b6a5ec95f9d7d67220ee228/zerver/lib/users.py) we don't know the admin. Also when we call ` check_change_full_name`  function we also don't know admin.